apple/Sources/Sargon/Extensions/Methods/Prelude/Bytes/Entropy16Bytes+Wrap+Functions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alexander Cyon on 2024-04-25.
+//
```

Comment:
should we strip this with Swiftformat?

Reply:
yes, just had not had time yet, I welcome PRs :)